joomla/joomla-cms
# [4.0] Edit article via button Edit in menu item single article

Pull Request for Issue # https://github.com/joomla/joomla-cms/issues/19316.

### Summary of Changes
In the file [/joomla-cms/administrator/components/com_content/Field/Modal/ArticleField.php](https://github.com/joomla/joomla-cms/blob/57a4fdecaf88794f54b80d7406572bd446a86dc7/administrator/components/com_content/Field/Modal/ArticleField.php#L96) we use JavaScript (document.getElementById("' . $this->id . '_id").value) in the edit URL. But up to now this command is not executed.
I am not sure where this is planned. This PR is my first idea for solving the error.


### Testing Instructions
See https://github.com/joomla/joomla-cms/issues/19316

Steps to reproduce the issue


- Installed staging 4.
- Installed testing data.
- Open "All Menu Items".
- Open menu item "Home" (Edit view). Type is : "Single Article"
- Click button "Edit" for editing the article "Joomla! Testing" .
  

### Expected result

Edit form opens with correct content.

### Actual result

Edit form opens but empty title and empty editor.

### Documentation Changes Required
No
